internal/telemetry/client.go

```diff
@@ -389,48 +427,115 @@ func (c *Client) newRequest(t RequestType) *Request {
 			LanguageVersion: runtime.Version(),
 		},
 		Host: Host{
-			Hostname:    hostname,
-			ContainerID: internal.ContainerID(),
-			OS:          getOSName(),
-			OSVersion:   getOSVersion(),
+			Hostname:  hostname,
+			OS:        getOSName(),
+			OSVersion: getOSVersion(),
+			// TODO (lievan): arch, kernel stuff?
```

Comment:
Anything that needs to happen in this PR?

Reply:
Just added architecture info to hostname, i think kernel name, release, and version can be saved for future work